weed/storage/store_ec.go

```diff
@@ -162,6 +162,32 @@ func (s *Store) ReadEcShardNeedle(vid needle.VolumeId, n *needle.Needle, onReadS
 	return 0, fmt.Errorf("ec shard %d not found", vid)
 }
 
+func (s *Store) ReadEcShardNeedleAt(vid needle.VolumeId, n *needle.Needle, offset int64, size int32) (int, error) {
```

Comment:
is it possible for me to pass a reader option that have flags that read meta data only